server/src/main/java/org/elasticsearch/action/admin/cluster/remote/RemoteInfoResponse.java

```diff
@@ -39,7 +39,7 @@
         infos = in.readList(RemoteConnectionInfo::new);
     }
 
-    RemoteInfoResponse(Collection<RemoteConnectionInfo> infos) {
+    public RemoteInfoResponse(Collection<RemoteConnectionInfo> infos) {
```

Comment:
I think all changes to the server gradle module can be undone now?

Reply:
It is used in `RemoteInfoResponseTests`.

Reply:
Ah! I missed that.